packages/user-profile-spa/README.md

```diff
@@ -0,0 +1,42 @@
+# Welcome to User Profile SPA 👋
+
+![Version](https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+
+User Profile SPA for One Platform
```

Comment:
Add some description about the User Profile SPA.

Reply:
Yes. @gisumit any suggestions ?